merged.graphql

```diff
@@ -40,7 +40,8 @@ input PluginMaster {
 	slack: SlackConfig
 	twilio: TwilioConfig
 	twitter: TwitterConfig
-	appgt: AppGTConfig
+	f_c_m: FCMConfig
```

Comment:
hm..I think the schema generator messed up the name for fcm. I can look into it.